src/v/cluster/inventory_service.h

```diff
@@ -127,6 +128,17 @@ class inventory_service : public ss::sharded<inventory_service> {
     ss::lowres_clock::duration _inventory_report_check_interval;
 
     bool _can_use_inventory_data{false};
+
+    // Controls if redpanda will create the report schedule on which the report
+    // is generated by cloud storage provider. If this value is false and the
+    // service is running, the schedule will not be created on service startup,
+    // but the report will still be checked for by the service in the expected
+    // path periodically. The expectation then is that the report will be
+    // manually placed in the bucket by the user. For use in testing and
+    // deployments where the cloud provider does not support inventory reports
+    // but reports can be generated externally, and inventory based scrub is
+    // still desired.
+    bool _should_create_report_config{true};
```

Comment:
nit: would it make sense for this to be a binding instead? Then we wouldn't require restart?

Reply:
This flag is a little tricky to support in terms of binding, it actually matters only for a very short amount of time. If this flag is false the configuration won't be created on startup, but the subsequent iterations of `check_for_current_inventory` won't re-read this variable to create the config (this is the current version of code, it is possible to change it so it is re-read each time). 

This is a setting which is not dynamic, either the backend supports scheduling, in which case we want this default IE true, or it doesn't, in which case it should always be false.

Even if this is true and the schedule already exists the service will handle it on startup, so we never need to set it to false unless it is a special backend like minio etc.